index.html

```diff
@@ -328,6 +328,10 @@
       <li>An <dfn>idle callback identifier</dfn>, which is a number which MUST
       initially be zero.</li>
     </ul>
+    <p>Each <a>IdleDeadline</a> has an [internal slot] which holds a
+    [DOMHighResTimeStamp] representing the absolute time in milliseconds of the
+    deadline. This slot MUST be populated when the <a>IdleDeadline</a> is
+    created.</p>
```

Comment:
You should give the internal slot a name. Something like "Each IdleDeadLine object has a [time] internal slot ..."